Thanks for the PR, unfortunately compilation of the test programs fails: 

```
wasm-ld: error: /project/bin/.emscripten_cache/sysroot/lib/wasm32-emscripten/lto/libnoexit.a(atexit_dummy.o): attempt to add bitcode file after LTO.
emcc: error: '/app/emsdk/upstream/bin/wasm-ld -o bin/assembly.wasm bin/assembly.bc -L/project/bin/.emscripten_cache/sysroot/lib/wasm32-emscripten/lto -lGL -lal -lhtml5 -lstubs -lnoexit -lc -ldlmalloc -lcompiler_rt -lc++-noexcept -lc++abi-noexcept -lsockets -mllvm -combiner-global-alias-analysis=false -mllvm -enable-emscripten-sjlj -mllvm -disable-lsr --import-undefined --strip-debug --export-if-defined=__test_MOD_add --export-if-defined=__test_MOD_dotest --export-if-defined=__lapack_prb_MOD_lapack_prb_main --export-if-defined=__test_eigen_MOD_test_eigen_main --export-if-defined=__start_em_asm --export-if-defined=__stop_em_asm --export=stackSave --export=stackRestore --export=stackAlloc --export=__wasm_call_ctors --export=__errno_location --export=malloc --export-table -z stack-size=5242880 --initial-memory=16777216 --no-entry --max-memory=16777216 --global-base=1024' failed (returned 1)
make: *** [bin/assembly.js] Error 1
makefile:99: recipe for target 'bin/assembly.js' failed
```

Did you clear your cache? (Delete `test/bin`, especially `test/bin/.emscripten_cache`)

--- a/test/makefile
+++ b/test/makefile
@@ -26,8 +26,8 @@ CC:=emcc
 WLD:=emcc
 FCFLAGS:=$(DEBUG) $(OPT) -Wall
 CCFLAGS:=$(DEBUG) --target=wasm32-unknown-emscripten $(OPT) -c -flto -emit-llvm -m32 -Isrc -Wall -s WASM=1
-EMSFLAGS:=$(OPT) $(DEBUG) -m32 -Wall -flto -Wl,-u,atexit -Wl,-u,__cxa_atexit
-WLDFLAGS:=-s WASM=1 -s EXPORTED_RUNTIME_METHODS=ccall,cwrap -s MODULARIZE=1 -s ERROR_ON_UNDEFINED_SYMBOLS=1 -s 'EXPORT_NAME="createModule"'
+EMSFLAGS:=$(OPT) $(DEBUG) -m32 -Wall -flto
+WLDFLAGS:=-Wl,-u,atexit -Wl,-u,__cxa_atexit -s WASM=1 -s EXPORTED_RUNTIME_METHODS=ccall,cwrap -s MODULARIZE=1 -s ERROR_ON_UNDEFINED_SYMBOLS=1 -s 'EXPORT_NAME="createModule"'
 #  -s USE_SDL=2 -s LEGACY_GL_EMULATION=1
 
 # Project config

--- a/test/makefile
+++ b/test/makefile
@@ -26,7 +26,7 @@ CC:=emcc
 WLD:=emcc
 FCFLAGS:=$(DEBUG) $(OPT) -Wall
 CCFLAGS:=$(DEBUG) --target=wasm32-unknown-emscripten $(OPT) -c -flto -emit-llvm -m32 -Isrc -Wall -s WASM=1
-EMSFLAGS:=$(OPT) $(DEBUG) -m32 -Wall -flto
+EMSFLAGS:=$(OPT) $(DEBUG) -m32 -Wall -flto -Wl,-u,atexit -Wl,-u,__cxa_atexit
 WLDFLAGS:=-s WASM=1 -s EXPORTED_RUNTIME_METHODS=ccall,cwrap -s MODULARIZE=1 -s ERROR_ON_UNDEFINED_SYMBOLS=1 -s 'EXPORT_NAME="createModule"'
 #  -s USE_SDL=2 -s LEGACY_GL_EMULATION=1
 

--- a/test/makefile
+++ b/test/makefile
@@ -25,7 +25,7 @@ FC:=emfc.sh
 CC:=emcc
 WLD:=emcc
 FCFLAGS:=$(DEBUG) $(OPT) -Wall
-CCFLAGS:=$(DEBUG) --target=wasm32-unknown-emscripten $(OPT) -c -flto -emit-llvm -m32 -Isrc -Wall -s WASM=1
+CCFLAGS:=$(DEBUG) --target=wasm32-unknown-emscripten $(OPT) -c -flto -emit-llvm -m32 -Isrc -Wall
 EMSFLAGS:=$(OPT) $(DEBUG) -m32 -Wall -flto
 WLDFLAGS:=-Wl,-u,atexit -Wl,-u,__cxa_atexit -s WASM=1 -s EXPORTED_RUNTIME_METHODS=ccall,cwrap -s MODULARIZE=1 -s ERROR_ON_UNDEFINED_SYMBOLS=1 -s 'EXPORT_NAME="createModule"'
 #  -s USE_SDL=2 -s LEGACY_GL_EMULATION=1

